java-base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6-941
```

Comment:
Seems like this introduced a new tag?

Reply:
duh :(
This was missing the trailing " AS" that my search and replace expression contained.